### PR TITLE
Blog about custom jmespath functions

### PR DESCRIPTION
**Pull request by Vishal-Chdhry**
Signed-off-by: Vishal Choudhary <sendtovishalchoudhary@gmail.com>

## Related issue: 
None

https://kubernetes.slack.com/archives/C032MM2CH7X/p1673274883889049?thread_ts=1673097525.979879&cid=C032MM2CH7X

## Proposed Changes
Creates a blog about jmespath.

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

**Review by chipzoller — CHANGES_REQUESTED** [review]
Again, I think you can cut out the lengthy examples and just focus on use cases. You should just focus on use cases surrounding the new time-based filters and probably not attempt to discuss how Kyverno uses JMESPath, which might be a multi-part series.

**Comment by Vishal-Chdhry** [comment]
@chipzoller Is this right?

**Review by chipzoller — COMMENTED** [review]
No, it's not quite what I meant. You're still just listing out all the filters very similar to what the documentation does. You're not telling a story around why they exist and some practical ways they can be used. The purpose of a blog is to tell that story, practical use, and with a conversational style.

**Comment by Vishal-Chdhry** [comment]
@chipzoller okay, i think I got it!

File: content/en/blog/general/jmespath/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+### Time_to_cron
+
+In kyverno, we use cron in several places to perform job scheduling. The cron expression enables users to schedule tasks to run periodically at a specified date/time. And it's naturally a great tool for automating lots of process runs, which otherwise would require human intervention. We have added a `time_to_cron()` filter to generate a cron expression from a time. For example, `time_to_cron('2022-04-11T03:14:05-07:00')` results in the output `"14 3 11 4 1"`. This filter can be used to generate things like ClusterCleanupPolicy.
+
+### Time_truncate
+The `time_truncate()` filter is used to round the given time to the nearest duration which is specified in the arguments. For example,  `time_truncate('2023-01-12T17:37:00Z','1h')` results in the output `"2023-01-12T17:00:00Z"`. 
+
+### Time_utc
+#### Description
+The `time_utc()` filter takes in a time in RFC 3339 format with a time offset and presents the same time in UTC/Zulu. It can be used in pair with `time_parse()` to convert it to UTC.
```
Comment (chipzoller) [inline content/en/blog/general/jmespath/index.md:59]: What I mean is all of this. Blogs don't need to list all the filters out. This is what documentation is for. Blogs tell a story around a use case in a casual style.

File: content/en/blog/general/jmespath/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Custom JMESPath Filters in Kyverno
+
+In addition to the filters available in the upstream JMESPath library which Kyverno uses, there are also many new and custom filters developed for Kyverno's use found nowhere else. These filters augment the already robust capabilities of JMESPath to bring new functionality and capabilities which help solve common use cases in running Kubernetes. The filters endemic to Kyverno can be used in addition to any of those found in the upstream JMESPath library used by Kyverno and do not represent replaced or removed functionality.
```
Comment (chipzoller) [inline content/en/blog/general/jmespath/index.md:17]: This is just copied directly from the documentation.

**Comment by chipzoller** [comment]
Any updates here?

**Comment by Vishal-Chdhry** [comment]
@chipzoller yeah i made some changes, are they okay?

**Review by chipzoller — APPROVED** [review]

**Pull request merged**